### PR TITLE
Add link back to aha

### PR DESCRIPTION
**Pull request by sbeckeriv**
Dearest Reviewer,

Fine day to add some issues. It's best if the github issues know where
they come from. Knowing there roots will help them expand and move
forward in their issue life cycle.

Added url if resource has url
Updated specs
Updated minor version

Becker

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -41,7 +41,7 @@ GIT
 PATH
   remote: .
   specs:
-    aha-services (1.24.16)
+    aha-services (1.24.17)
```
Comment (schneidmaster) [inline Gemfile.lock]: Looks like this is `1.24.17` but the version.rb is `1.24.18` -- can you bundle again so they match?

File: lib/services/github_issues.rb [hunk 2]
```diff
@@ -256,6 +256,8 @@ def issue_body(resource)
     if resource.description.attachments.present?
       issue_body_parts << attachments_in_body(resource.description.attachments)
     end
+    issue_body_parts << "Created from Aha! #{resource.url}" if resource.respond_to?(:url)
```
Comment (k1w1) [inline lib/services/github_issues.rb]: How does this work with webhooks? If the description is updated in Github will it appear in Aha! with this link? Will the link be added a second time when it is send back to Github?

Comment (sbeckeriv) [inline lib/services/github_issues.rb]: It looks like updates in github are ignored on the aha side currently. The update command to github rewrites the issue body.

**Comment by sbeckeriv** [comment]
@schneidmaster add gitlab

**Pull request merged**